osu.Game/Screens/Play/HUD/HitErrorMeters/BarHitErrorMeter.cs

```diff
@@ -55,89 +56,112 @@ private void load(OsuColour colours)
                             {
-                                Y = 10,
                                 Size = new Vector2(10),
                                 Icon = FontAwesome.Solid.Bicycle,
-                                Anchor = Anchor.BottomCentre,
-                                Origin = Anchor.BottomCentre,
+                                Anchor = alignment == Anchor.x1 ? Anchor.CentreRight : Anchor.BottomCentre,
+                                Origin = alignment == Anchor.x1 ? Anchor.CentreRight : Anchor.BottomCentre,
+                                Y = alignment == Anchor.x1 ? 0 : 10,
+                                X = alignment == Anchor.x1 ? 10 : 0,
                             }
                         }
                     },
-                    new Container
+                    arrowContainer = new Container
                     {
                         Anchor = Anchor.y1 | alignment,
                         Origin = Anchor.y1 | alignment,
-                        Width = chevron_size,
-                        RelativeSizeAxes = Axes.Y,
+                        RelativeSizeAxes = alignment == Anchor.x1 ? Axes.X : Axes.Y,
                         Child = arrow = new SpriteIcon
                         {
-                            Anchor = Anchor.TopCentre,
-                            Origin = Anchor.Centre,
-                            RelativePositionAxes = Axes.Y,
-                            Y = 0.5f,
-                            Icon = alignment == Anchor.x2 ? FontAwesome.Solid.ChevronRight : FontAwesome.Solid.ChevronLeft,
+                            Anchor = alignment == Anchor.x1 ? Anchor.BottomCentre : Anchor.TopCentre,
+                            Origin = alignment == Anchor.x1 ? Anchor.BottomCentre : Anchor.Centre,
+                            RelativePositionAxes = alignment == Anchor.x1 ? Axes.X : Axes.Y,
+                            Icon = alignment == Anchor.x1 ? FontAwesome.Solid.ChevronUp : (alignment == Anchor.x2 ? FontAwesome.Solid.ChevronRight : FontAwesome.Solid.ChevronLeft),
                             Size = new Vector2(chevron_size),
                         }
                     },
                 }
             };
 
+            if (alignment == Anchor.x1)
+            {
+                InternalChild.Width = bar_height;
+                judgementsContainer.Height = judgement_line_width;
+                colourBars.Height = bar_width;
+                colourBarsEarly.Width = 0.5f;
+                colourBarsLate.Width = 0.5f;
+                arrowContainer.Height = chevron_size;
+            }
+            else
+            {
+                InternalChild.Height = bar_height;
+                judgementsContainer.Width = judgement_line_width;
+                colourBars.Width = bar_width;
+                colourBarsEarly.Height = 0.5f;
+                colourBarsLate.Height = 0.5f;
+                arrowContainer.Width = chevron_size;
+                arrow.Y = 0.5f;
+            }
```

Comment:
There are also other checks like here for example. just to make it clear, `Width` and `Height` are components of the vector `Size`, `X` and `Y` are components of `Position`, so you can apply the same thing as I said above.

Reply:
Didn't know these were shorthand properties for accessing Size & Position properties